travel-event-host-app/src/lib/category.ts

```diff
@@ -0,0 +1,7 @@
+enum Category {
+  category1 = 'category 1',
+  category2 = 'category 2',
+  category3 = 'category 3',
+  category4 = 'category 4',
```

Comment:
Can we use similar categories to meetup since the data will be more meaningful?
```suggestion
  TravelOutdoor= 'travelOutdoor',
  SocialActivities= 'socialActivities',
  HobiesPassions= 'hobbiesPassions',
...etc
```